runtimevar/blobvar/example_test.go

```diff
@@ -57,25 +47,28 @@ func ExampleOpenVariable() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	// runtimevar.Snapshot.Value is decoded to type MyConfig.
-	cfg := snapshot.Value.(MyConfig)
-	fmt.Printf("%s running on port %d", cfg.Server, cfg.Port)
+	// runtimevar.Snapshot.Value is decoded to a string.
+	fmt.Println(snapshot.Value.(string))
 
 	// Output:
-	// foo.com running on port 80
+	// hello world
 }
 
 func Example_openVariableFromURL() {
+	// This example is used in https://gocloud.dev/howto/runtimevar/runtimevar/#blob
+
+	// import _ "gocloud.dev/runtimevar/blobvar"
+
 	// runtimevar.OpenVariable creates a *runtimevar.Variable from a URL.
 	// The default opener opens a blob.Bucket via a URL, based on the environment
 	// variable BLOBVAR_BUCKET_URL.
-	// This example watches a JSON variable.
+
+	// Variables set up elsewhere:
```

Comment:
I think Ross was trying to make the examples as short as possible. I don't feel strongly about it, but I think it's consistent across the examples that initializations for variables like `ctx` is hidden on `gocloud.dev`.

Yes, these comments also show up in the `godoc`. I don't think there's a great way to avoid that, although maybe the comments could be more obvious that they are directives and should be ignored by humans.

Reply:
Done: https://github.com/google/go-cloud/issues/2582.